SWLOR.Game.Server/Service/RoleplayService.cs

```diff
@@ -97,13 +97,13 @@ private static void ProcessPlayerRoleplayXP(NWPlayer player)
             var dbPlayer = DataService.Player.GetByID(player.GlobalID);
             if (dbPlayer.RoleplayPoints >= 50)
             {
-                const int BaseXP = 500;
+                const int BaseXP = 1000;
                 float residencyBonus = PlayerStatService.EffectiveResidencyBonus(player);
                 int xp = (int)(BaseXP + BaseXP * residencyBonus);
                 float dmBonusModifier = dbPlayer.XPBonus * 0.01f;
                 if (dmBonusModifier > 0.25f)
                     dmBonusModifier = 0.25f;
-                xp += (int)(xp * dmBonusModifier);
+                xp += (int)(xp * dmBonusModifier + (dbPlayer.RoleplayPoints * 50));
```

Comment:
Purpose of this change?

Reply:
I added this as a way that adds some variability into the numbers you get. Rewarding players who RP more over those that would barely hit the cap as there was none.  It is a flat amount (and now thinking on it that math is wrong is should be  ((dbPlayer.RoleplayPoints - 50) * 50). So you basically get 50 xp for every line over the original trigger. it is added after all other calculations so it doesn't get boosted or anything.

Reply:
Yea this is adding a minimum of 2500 on every yield. So that's at a minimum of 3500, before XP bonuses are applied. If you swap it to ((dbPlayer.RoleplayPoints - 50) * 50) so that only the excess points count as XP, I can get behind this change.